pkg/cmd/openshift-operators/util/resourceapply/core.go

```diff
@@ -94,10 +94,7 @@ func ApplyConfigMap(client coreclientv1.ConfigMapsGetter, required *corev1.Confi
 
 	modified := resourcemerge.BoolPtr(false)
 	resourcemerge.EnsureObjectMeta(modified, &existing.ObjectMeta, required.ObjectMeta)
-	if !*modified {
-		return false, nil
-	}
-	if equality.Semantic.DeepEqual(existing.Data, required.Data) {
+	if equality.Semantic.DeepEqual(existing.Data, required.Data) && !*modified {
```

Comment:
heh, I tested updating the instance config first, guess this is what was broken :)